{{cookiecutter.project_name}}/monitoring/README.md

```diff
@@ -0,0 +1,63 @@
+# Setup Model Monitoring
```

Comment:
Should we consistently refer to it as `Data Monitoring` everywhere? That is the current name of the product, so it might be misleading to see model some places and data in others.

Reply:
Data monitoring is for general tables. This notebook is specifically for monitoring the inference log table.
There could be feature monitoring or other table monitoring in the future. Are you suggesting maybe unify them?

Reply:
data monitoring for inference table